templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-service-guacamole-windowsvm
-version: 1.2.0
+version: 1.2.2
```

Comment:
1.2.1?

Reply:
In my environment I had to move it up another one...